packages/aws/data_stream/billing/manifest.yml

```
@@ -13,6 +13,7 @@ streams:
       - name: data_granularity
         type: text
         title: Data Granularity
+        description: "Granularity of the data points returned in the Collection Period. Example values are `1m` or `5m`, see Golang [time.ParseDuration](https://pkg.go.dev/time#ParseDuration) for more details. Allows you to query CloudWatch asking “Give me the four 5-minute (Data Granularity) data points for the last 20 minutes (Collection Period)“. To learn more about how to use data granularity, see [Metrics collection and cost considerations](https://docs.elastic.co/integrations/aws#metrics-collection-and-cost-considerations)."
```

I agree and this is a hard one to explain. What do you think about this @zmoog @girodav : 
```suggestion
        description: Granularity of the returned data points. Example values are `1m` , `5m` and etc, see Golang [time.ParseDuration](https://pkg.go.dev/time#ParseDuration) for more details. This parameter works with `period` together to define how frequently the metrics are collected and what the time difference is between returned data points. `data_granularity` value cannot be lower than the frequency CloudWatch publishes events and also cannot be higher than `period`. To learn more about how to use data granularity, see [Metrics collection and cost considerations](https://docs.elastic.co/integrations/aws#metrics-collection-and-cost-considerations).
```